### PR TITLE
Create database and query

### PR DESCRIPTION
**Pull request by dazehinn**
- create database
- create animals table
- populate animals table
- query the animals table (see screenshots below)
  

- [ ] all animals whose name ends in "mon".
01_end_in_mon

- [ ] name of all animals born between 2016 and 2019.
02_dob_2016_2019

- [ ] name of all animals that are neutered and have less than 3 escape attempts.
03_neutered_ _3_attempts

- [ ] date of birth of all animals named either "Agumon" or "Pikachu".
04_dob_Agumon_Pikachu

- [ ] name and escape attempts of animals that weigh more than 10.5kg
05_more_than_10 5kg

- [ ] all animals that are neutered.
06_neutered_animals

- [ ] all animals not named Gabumon.
07_not_named_Gabumon

- [ ] all animals with a weight between 10.4kg and 17.3kg
08_weigth_btn_10 4_17 3

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-    name varchar(100)
+CREATE DATABASE vet_clinic;
+
+CREATE table animals (
+  id int,
+  name text,
+  date_of_birth date,
+  escape_attempts int,
+  neutered boolean,
+  weight_kg real
```
Comment (Ghiftee) [inline schema.sql]: - [ REQUIRED ] Kindly set the data type of ```name to varchar``` and ```weight_kg to decimal``` to match the requirements.
- [ OPTIONAL ] You can consider marking the id column as primary key, not null and autoincremented, as it ensures uniqueness, data integrity, referential integrity, indexing, and can enhance performance. e.g. ``id int NOT NULL IDENTITY(1,1)`` `` PRIMARY KEY (id)``

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals
+    (name, date_of_birth, weight_kg, neutered, escape_attempts)
+  VALUES 
+    ('Agumon', '2020-02-03', 10.23, true, 0),
+    ('Gabumon', '2018-11-15', 8, true, 2),
+    ('Pikachu','2021-01-01', 15.04, false, 1),
```
Comment (Ghiftee) [inline data.sql]: - [ REQUIRED ] Kindly ensure that this date of birth is correct, it should be `2021/01/07`

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @dazehinn 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Commands are in the correct files
✔️ Queries are successful
✔️ Screenshots are added

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @dazehinn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  Fair README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (dazehinn) [reply schema.sql]: thanks for the suggestion on the primary key

**Pull request merged**